jenkinsfiles/create_user.groovy

```diff
@@ -13,6 +13,6 @@ node {
     }
 
     stage ("Create platform user") {
-        sh "jenkinsfiles/create_user.sh ${env.PLATFORM_ENV} ${env.USERNAME} ${env.EMAIL} \"${env.FULLNAME}\""
+        sh "/usr/local/bin/create_user ${env.USERNAME} ${env.EMAIL} --env ${env.PLATFORM_ENV} --fullname \"${env.FULLNAME}\""
```

Comment:
Looks good to me. I had a question about the use of `--` for all these arguments in the other PR: https://github.com/ministryofjustice/analytics-platform-jenkins-agent/pull/4/files#r120372836